enterprise/models.py

```diff
@@ -316,8 +316,9 @@ class Meta:
         verbose_name="Disable expiration messaging for learner credit",
         default=False,
         help_text=_("If checked, learners and admins will not receive notifications leading up to the expiration "
-                    "date for learner credit plans. Notifications signaling the expiration (and loss of usability) "
-                    "itself will still appear.")
+                    "date for learner credit plans. This includes removing the expiration date from the subsidy "
+                    "box for upcoming expiratons. The subsidy summary box will not display for expired plans. "
+                    "Other, notifications signaling the expiration (and loss of usability) itself will still appear.")
```

Comment:
Nit: stray comma after "Other".  Whether you want to regen migrations to fix this is up to you :p

Reply:
Good catch. Regened 👍🏽